src/browser/api/window.js

```diff
@@ -1561,7 +1561,8 @@ Window.getZoomLevel = function(identity, callback) {
 Window.setZoomLevel = function(identity, level) {
     let browserWindow = getElectronBrowserWindow(identity, 'set zoom level for');
 
-    browserWindow.webContents.setZoomLevel(level);
+    // browserWindow.webContents.setZoomLevel(level); // zooms all windows loaded from same domain
```

Comment:
Is this commented out code intentional?

Reply:
yes, as a caution to future maintainers not to revert to calling `setZoomLevel` directly, which gives the wrong result